### PR TITLE
feat: implement "Container" to connect "PortexType" and data container

### PR DESCRIPTION
**Pull request by linjiX**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-29xa0z2 Portex Schema 支持拓展类型](https://app.clickup.com/t/29xa0z2)

File: graviti/dataframe/frame.py [hunk 1]
```diff
@@ -151,43 +152,27 @@ def _get_repr_header(flatten_header: List[Tuple[str, ...]]) -> List[List[str]]:
...
-                obj._columns[column_name] = DataFrame._from_pyarrow(
-                    column_value, schema=column_schema  # type:ignore[arg-type]
-                )
-            else:
-                obj._columns[
-                    column_name
-                ] = ColumnSeries._from_pyarrow(  # pylint: disable=protected-access
-                    column_value, schema=column_schema
-                )
+        # In this case schema.to_builtin() should always record.
```
Comment (Lee-000) [inline graviti/dataframe/frame.py:168]: `In this case schema.to_builtin() always returns record`?

File: graviti/dataframe/column/series.py [hunk 2]
```diff
@@ -11,15 +11,27 @@
...
-from graviti.portex import PortexType
+from graviti.dataframe.container import Container, ContainerRegister
 from graviti.utility import MAX_REPR_ROWS
 from graviti.utility.paging import PagingList
 
 _T = TypeVar("_T", bound="Series")
 
 
-class Series:
+@ContainerRegister(
```
Comment (Lee-000) [inline graviti/dataframe/column/series.py:23]: Only builtin types?

Comment (linjiX) [reply graviti/dataframe/column/series.py:23]: I will implement another register for extension type

File: graviti/dataframe/frame.py (hunk 1 above)
Comment (linjiX) [reply graviti/dataframe/frame.py:168]: if the check in `DataFrame.from_pyarrow` passed, the `schema.to_builtin` should always be `record`

Comment (Lee-000) [reply graviti/dataframe/frame.py:168]: I mean should we rephrase the sentence to the previous comment?

**Review by Lee-000 — APPROVED** [review]

**Review by AChenQ — APPROVED** [review]

**Pull request merged**